.github/workflows/reusable-build-test-release.yml

```diff
@@ -2111,46 +2142,54 @@ jobs:
           git_committer_email: ${{ secrets.SA_GH_USER_EMAIL }}
           gpg_private_key: ${{ secrets.SA_GPG_PRIVATE_KEY }}
           passphrase: ${{ secrets.SA_GPG_PASSPHRASE }}
+      - name: Release custom version
+        if: ${{ github.event.inputs.custom-version  != '' }}
+        uses: "softprops/action-gh-release@v2"
+        with:
+          token: "${{ secrets.GH_TOKEN_ADMIN }}"
+          tag_name: v${{ github.event.inputs.custom-version }}
+          target_commitish: "${{github.ref_name}}"
+          make_latest: false
       - name: Download package-deployment
-        if: ${{ steps.semantic.outputs.new_release_published == 'true' }}
+        if: ${{ steps.semantic.outputs.new_release_published == 'true' || github.event.inputs.custom-version  != ''}}
```

Comment:
Could we rather rely on some output of step Release custom version?

Reply:
Resolved - now checking if `upload_url` output exists for the custom release step